### PR TITLE
add Nicaragua

### PR DESCRIPTION
**Pull request by jarek**
See comments in parser script for sources and details.

### CODE REVIEW

File: parsers/NI.py [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+
+    # We're currently using the summary page (SUMMARY_URL, via get_production_from_summary())
+    # rather than the detailed map page (MAP_URL, via get_production_from_map())
+    # in order to get solar production.
+    production, data_datetime = get_production_from_summary(requests_obj)
+
+    data = {
+        'datetime': data_datetime,
+        'countryCode': country_code,
+        'production': production,
```
Comment (corradio) [inline parsers/NI.py]: Are you able to always set some production keys to 0? For example coal, if we know NI does not have coal power plants?

File: parsers/config.py [hunk 2]
```diff
@@ -199,6 +201,9 @@
     # 'MD->RO':     ENTSOE.fetch_exchange,
     # MK
     'MK->RS':     ENTSOE.fetch_exchange,
+    # NI
+    'CR->NI':     NI.fetch_exchange,
```
Comment (corradio) [inline parsers/config.py]: I tend to sort alphabetically, meaning that `CR->NI` should be under`CR`. Same for `HN->NI` which should be under `HN`

**Comment by corradio** [comment]
Looking awesome!
However, it seems like arrows need to be rotated by 180deg:

image

File: parsers/config.py (hunk 2 above)
Comment (jarek) [inline parsers/config.py]: Ah sure, I noticed that trend when adding it to web app exchanges config, but forgot it here. Will fix.

File: parsers/NI.py (hunk 1 above)
Comment (jarek) [inline parsers/NI.py]: Yep sure! I wasn't sure how important this was when there was already a mix of a few modes. I'll add it.

**Comment by jarek** [comment]
Fixed the exchange arrows and the other requests. Yesterday I thought out the exchange arrows very carefully, understanding how the arrow-drawing works and thinking which direction they should be in... and in the end managed to get them exactly opposite ;)

**Comment by corradio** [comment]
Yes it's a bit tricky I must admit :)

**Pull request merged**